pkg/cvo/cvo.go

```diff
@@ -93,6 +93,9 @@ type Operator struct {
 	// minimumUpdateCheckInterval is the minimum duration to check for updates from
 	// the upstream.
 	minimumUpdateCheckInterval time.Duration
+	// architecture identifies the architecture of the cluster nodes. The cluster architecture
+	// is heterogeneous if the cluster has nodes running on different architectures.
```

Comment:
Can we add about the possible value of architecture when cluster is heterogeneous i.e. `multi` and for others it would be `GOARCH`.

Reply:
if we want to doc this, I think we should just say "to match the Cincinnati API's 'arch' property" or something, and then update [this][1] in  openshift/cincinnati#711 to cover `multi` as well.

[1]: https://github.com/openshift/cincinnati/blob/master/docs/design/openshift.md#request

Reply:
I am fine with that as long as we can find out from the doc what values can be used for it.

Reply:
Updated comment